docs/investigations/api-documentation/generate-api-documentation.md

```diff
@@ -0,0 +1,59 @@
+# Generating API documentation
+
+## Overview
+
+This document discusses how to create a static documentation page from our GraphQL schema.
+
+As of right now, there is no static page with our API documentation, so if any problem occurs the user can either check the `examples` directory or read the schema on his own.
+We need a solution that would be available on one of the Compass endpoints and would utilise our generated examples.
+
+## Possible solutions
+
+### 1. Links to the examples on the API playground
+The GraphQL playground that we use on our endpoints generates documentation with comments from the schema.
+We could create a gqlgen plugin that would add a relative example link to every query and mutation. That way the user
+would be able to check how the request looks right away.
+
+Work that has to be done
+* gqlgen plugin
+* hosting and serving the examples on the director
```

Comment:
another point for this section:

make sure that GraphQL Playground page (GET request) is not secured for every type of authentication